.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,105 @@
+name: Integration Tests
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+
+jobs:
+
+  # We borrow the tests from the Google I/O Android App under the Apache license: https://github.com/google/iosched/.
+  # The following steps checkout iosched and apply a github patch to the project. The patch adds
+  # Sentry to the app with auto performance monitoring enabled. We then run the UI tests to make sure
+  # adding our SDK doesn't cause any major issues.
+  build-iosched:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'google/iosched'
+          ref: '6f96c1528c00d3d89bedb33e5a0a10d67629e8bd'
+
+      - uses: actions/checkout@v2
+        with:
+          path: 'sentry-android-gradle-plugin'
+
+      - name: Add Sentry
+        run: git apply ./sentry-android-gradle-plugin/scripts/add-sentry-to-iosched.patch
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: iosched-gradle-${{ hashFiles('checksum.txt') }}
```

Comment:
Should you use the same Gradle setup @romtsn introduced?

Reply:
I just copied this from https://github.com/google/iosched/blob/6f96c1528c00d3d89bedb33e5a0a10d67629e8bd/.github/workflows/iosched.yaml#L30-L36. I'm going to add a comment to avoid confusion in the future.